linera-views/src/dynamo_db.rs

```diff
@@ -191,6 +203,20 @@ fn extract_key_value_owned(
     Ok((key, value))
 }
 
+/// This computes the offset of the serialization of a vector.
+/// The formula that should be satisfied is
+/// serialized_size(vec![v_1, ...., v_n]) = get_bitsize(n)
+///  + serialized_size(v_1)? + .... serialized_size(v_n)?
+pub fn get_bitsize(len: usize) -> usize {
```

Comment:
Not sure about the name: It returns the number of bytes, not bits (but the standard library's `size_of` also does, implicitly, so maybe no need to specify); and it might be helpful to clarify that it's about the ULEB128 encoding of a number.

```suggestion
pub fn get_uleb128_size(len: usize) -> usize {
```